### PR TITLE
Update pootle_store.models unit.save

### PR DESCRIPTION
**Pull request by phlax**
improve handling of change vars

### CODE REVIEW

File: pootle/apps/pootle_store/models.py [hunk 1]
```diff
@@ -308,21 +310,26 @@ def save(self, *args, **kwargs):
...
+                or not self.change.submitted_on)
+            if update_submit:
+                self.change.submitted_by = submitted_by or sysuser
                 self.change.submitted_on = submitted_on
-            if reviewed_on is not None:
-                self.change.reviewed_on = reviewed_on
+            if reviewed_by is not None:
+                self.change.reviewed_by = reviewed_by or submitted_by
+            if reviewed_on is not None or reviewed_by is not None:
+                self.change.reviewed_on = reviewed_on or submitted_on
```
Comment (unho) [inline pootle/apps/pootle_store/models.py]: If `reviewed_on` is not `None` is the `or submitted_on` actually necessary?

Comment (phlax) [inline pootle/apps/pootle_store/models.py]: yes if only reviewed_by is set

Comment (unho) [inline pootle/apps/pootle_store/models.py]: I feel that a bit odd, but wfm.

**Comment by unho** [comment]
lgtm

**Pull request merged**